app/src/components/HomePage.vue

```diff
@@ -27,41 +31,64 @@
 }
 
 .submit-search-button {
-  background: #DD1C1A;
+  background: $button-color;
   border: none;
   border-radius: 25px;
   box-shadow: 0px 2px 4px 0px rgba(0,0,0,0.50);
-  color: #FFF1D0;
+  color: $button-text;
   font-size: 20px;
   margin-top: 15px;
   outline: none;
   padding: 10px;
   width: 120px;
   &:hover {
-    background: darken(#DD1C1A, 15%);
+    background: darken($button-color, 15%);
     cursor: pointer;
     transition: 0.4s;
   }
 }
 
 .modal-container {
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    position: fixed;
-    z-index: 9998;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    background-color: rgba(0, 0, 0, .5);
-    transition: opacity .3s ease;
+  align-items: center;
+  background-color: rgba(0, 0, 0, .5);
+  display: flex;
+  justify-content: center;
+  height: 100%;
+  left: 0;
+  position: fixed;
+  top: 0;
+  transition: opacity .3s ease;
+  width: 100%;
+  z-index: 1;
```

Comment:
No problem, I like cleaning up code 👌 And I know this is a strange thing to like, but I like putting the styling in alphabetical order.